Tools/cases_generator/generate_cases.py

```diff
@@ -15,14 +15,14 @@
 import parser
 from parser import StackEffect
 
-DEFAULT_INPUT = os.path.relpath(
-    os.path.join(os.path.dirname(__file__), "../../Python/bytecodes.c")
-)
-DEFAULT_OUTPUT = os.path.relpath(
-    os.path.join(os.path.dirname(__file__), "../../Python/generated_cases.c.h")
-)
+HERE = os.path.dirname(__file__)
+ROOT = os.path.join(HERE, "../..")
```

Comment:
```suggestion
ROOT = os.path.join(HERE, os.pardir, os.pardir)
```

Reply:
You know that Windows allows forward slashes as well right? :-)

Reply:
I keep forgetting that. Does it allow mixing them in the same path?

In that case, I guess I don't really see the advantage of `os.path.join` over string concatenation. Maybe to avoid duplicating path separators on accident? At any rate, feel free to ignore these suggestions then.

Reply:
1. Yes.
2. I never use string ops for path joining/splitting, but I am comfortable with literal paths of multiple segments.
3. I will ignore this suggestion.